src/__experimental__/components/input-icon-toggle/input-icon-toggle-classic.style.js

```diff
@@ -0,0 +1,26 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../style/themes';
+
+export default ({
+  inputFocus, inputHover, theme, type
+}) => theme.name === THEMES.classic && css`
+  background-color: #e6ebed;
+  border-left: 1px solid #bfccd2;
+  box-sizing: border-box;
+  color: #003349;
+  cursor: pointer;
+  margin-right: -6.5px;
+  width: ${type === 'dropdown' ? '20px' : '31px'};
+
+  ${(inputHover || inputFocus) ? '' : '&:hover'} { color: #fff };
+
+  ${inputFocus ? '' : '&:hover'} {
+    background-color: #004b87;
+    border-color: #004b87;
+  }
+
+  ${inputHover && css`
+    background-color: #1963f6;
+    border-color: #1963f6;
+  `}
+`;
```

Comment:
all these colors should be defined in the classic theme

Reply:
All of the colors in the classic theme config are shades of green, which isn't correct here based on the demo site styles - I can update them, but was hesitant to do so in case it broke something else

Reply:
you should add those colors to `classic-theme.config.js`